core/load-git.c

```diff
@@ -735,6 +744,8 @@ static void parse_event_keyvalue(void *_event, const char *key, const char *valu
 		event->value = val;
 	} else if (!strcmp(key, "name")) {
 		/* We get the name from the string handling */
+	} else if (!strcmp(key,"divemode")) {
+		event->value = get_divemode(value);
 	} else if (!strcmp(key, "cylinder")) {
```

Comment:
this has the `{ }` around `if / then` one liner but it was there before, i guess.